### ISSUE TITLE
Finetune the pretrained embedding model

### ISSUE THREAD

**Issue by legendzzy** [anton-bushuiev/PPIformer#8]
Hi! I see that you have provided a pretrained model for PPI embedding. I wonder if it is possible to finetune the PPI embedding model with other PDB data and label directly?

**Comment by anton-bushuiev** [comment]
Hi @legendzzy, unfortunately there is no code for directly fine-tuning the model for a custom classification/regression task. I suggest you could try extracting embeddings for your data with the frozen model (as in the README example), and then using the output tensors in your custom code. If it works well, it may be worth looking into how to fine-tune the model to improve the performance further.

**Comment by anton-bushuiev** [comment]
Feel free to reopen.
